Geometry/HGCalGeometry/test/HGCalWaferInFileTest.cc

```diff
@@ -0,0 +1,305 @@
+// -*- C++ -*-
+//
+// Package:    HGCalWaferInFileTest
+// Class:      HGCalWaferInFileTest
+//
+/**\class HGCalWaferInFileTest HGCalWaferInFileTest.cc
+ test/HGCalWaferInFileTest.cc
+
+ Description: <one line class summary>
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+// Original Author:  Sunanda Banerjee
+//         Created:  Mon 2020/06/24
+//
+//
+
+// system include files
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
```

Comment:
Looks like `FWCore/ParameterSet` and `Geometry/HGCalGeometry` should be added to `Geometry/HGCalCommonData/test/BuildFile.xml`